src/routes/models/scheme-payments-by-year/fetch.js

```
@@ -0,0 +1,11 @@
+import { getSchemePaymentsByYear } from '../../../api/get-scheme-payments-by-year.js'
+
+export async function fetchSchemePaymentsByYear () {
+  const schemePaymentsByYear = await getSchemePaymentsByYear()
+
+  if (!schemePaymentsByYear) {
+    throw new Error()
```

Do we want to throw an error here? Or do we want to use the CPD logger?